ui/client/containers/hooks/useSearchQuery.tsx

```diff
@@ -0,0 +1,24 @@
+import {defaults} from "lodash"
+import * as queryString from "query-string"
```

Comment:
You used two times import..

Reply:
has to be this way, one for commonjs module, one for type